partEZ/app/Http/Controllers/EventController.php

```diff
@@ -456,6 +474,17 @@ public static function sendInvitation($eid, $email, $uid)
         });
     }
 
+    public static function sendCancellation($eid, $email)
+    {
```

Comment:
I agree with you, and we actually do have one, but I wasn't ever able to get it to work properly.